app/models/concerns/sequenced.rb

```diff
@@ -55,6 +60,14 @@ def sequenced_options=(options)
     def sequenced_options
       @sequenced_options
     end
+
+    def sequenced_lock_key=(lock_key)
+      @sequenced_lock_key = lock_key
+    end
+
+    def sequenced_lock_key
+      @sequenced_lock_key
```

Comment:
ThreadSafety/InstanceVariableInClassMethod: Avoid instance variables in class methods.